src/__tests__/index.js

```diff
@@ -254,5 +254,16 @@ pluginTester({
         }
       },
     },
+    {
```

Comment:
Could you verify for me that this test fails without the changes in the source? Maybe take a screenshot?

Reply:
Never mind! You thought ahead!

> First I'll push without the fix just to see the tests fail.
> Just after that I'll push the fix.

Thanks!